Dockerfile

```diff
@@ -13,6 +13,7 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -installsuffix cgo -ldflag
 FROM scratch
 COPY --from=builder /build/rageshake /rageshake
 COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+COPY --from=builder /build/rageshake.sample.yaml /rageshake.yaml
```

Comment:
I'm not convinced this makes much sense. The sample config file isn't ready for use without customisation.

Reply:
Yes I do agree.. but no reference as to that it should be renamed to rageshake.yaml is anywhere in the install guide or so. Hence I just made the customisations and left the file as is.

Reply:
But concise information is given that the customisations should be done in the rageshake.sample.yaml